index.js

```diff
@@ -682,10 +682,29 @@
   //. > S.maybe(0, R.length, S.Nothing())
   //. 0
   //. ```
-  S.maybe = def('maybe', [a, Function, Maybe], function(x, f, maybe) {
+  var maybe = S.maybe =
+  def('maybe', [a, Function, Maybe], function(x, f, maybe) {
     return fromMaybe(x, maybe.map(f));
   });
 
+  //  catMaybes :: [Maybe a] -> [a]
+  var catMaybes = def('catMaybes', [Accessible], R.chain(maybe([], R.of)));
+
+  //# mapMaybe :: (a -> Maybe b) -> [a] -> [b]
+  //.
+  //. Takes a function and a list, and returns a list containing the value
+  //. of each Just resulting from the application of the function to each
+  //. element of the input list.
+  //.
```

Comment:
This description was a little confusing to me at first.. maybe mention that `Nothing`s are rejected in the description?